src/components/menuAppBar/MenuAppBar.tsx

```diff
@@ -35,28 +47,39 @@ const MenuAppBar = () => {
         alt="Go to Scalac homepage"
       />
       <div className="AppBar__menu-btn-container">
-        <button type="button" onClick={() => goTo("/")}>
-          <img src="/dashboard.svg" alt="" />
-          Convert
-        </button>
-        <button type="button" onClick={() => goTo("/contact")}>
-          <img src="/mail.svg" alt="" />
-          Contact
-        </button>
-
+        <NavButton
+          name={"Convert"}
+          goTo={() => goTo(routes.home)}
```

Comment:
Why are you passing goTo? You can import router inside btn and create this function on your own.